os-server/src/main/java/app/openschool/usermanagement/api/dto/UserRegistrationHttpResponse.java

```diff
@@ -0,0 +1,84 @@
+package app.openschool.usermanagement.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.util.Date;
+import java.util.HashMap;
+import org.springframework.http.HttpStatus;
+
+/** Useful Javadoc. */
+public class UserRegistrationHttpResponse {
+
+  @JsonFormat(
+      shape = JsonFormat.Shape.STRING,
+      pattern = "MM-dd-yyyy hh:mm:ss",
+      timezone = "Asia/Yerevan")
+  private Date timeStamp;
+
+  private int httpStatusCode;
+  private HttpStatus httpStatus;
```

Comment:
you shouldn't return info about the HTTP status code in the body, it can be returned as part of HttpResponseEntity

Reply:
please remove the status as well